src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1188,7 +1189,10 @@ hypre_MGRSetup( void               *mgr_vdata,
                                                    -1, CF_marker, inv_size, 1, diag_inv);
             frelax_diaginv[lev] = diag_inv;
             blk_size[lev] = block_jacobi_bsize;
-            hypre_MGRBuildAff(A_array[lev], CF_marker, debug_flag, &A_FF);
+            if (!A_FF)
```

Comment:
Was this a bug before? Could AFF have been build previously?

Reply:
Right, this was a bug before that our tests didn't catch